projects/media-viewer/src/lib/annotations/annotation-set/annotation-create/box-highlight-create.component.spec.ts

```diff
@@ -132,7 +132,7 @@ describe('BoxHighlightCreateComponent', () => {
     component.top = 50;
     component.left = 50;
 
-    component.create();
+    component.create(1);
```

Comment:
not sure if this create is declarative enough maybe createHightlight would make more sense also 1 maybe stick ti into a var called page

Reply:
I'll update it